Scripts/SL-PlayerOptions.lua

```diff
@@ -396,6 +396,20 @@ local Overrides = {
 
 			-- stringify(range(first, last, step), "%g") causes 0 to display as a small exponential
 			-- so we have to use %.3f and remove trailing zeroes with gsub and a bit of magic (regexp)
+			local t = {}
+			for k, v in pairs(range(first, last, step)) do
+				earlylate = " (notes come "..( v > 0 and "earlier)" or "later)" )
```

Comment:
Let's remove "come" to make it easier to read at a glance?

```suggestion
				earlylate = " (notes "..( v > 0 and "earlier)" or "later)" )
```